.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: go
+go:
+  - "1.12"
+env:
+- GO111MODULE=on
+script:
+  - make test
```

Comment:
If you choose to add this file, there's also a brief guide [here](https://travis-ci.org/getting_started?utm_medium=notification&utm_source=github_status) on the other step needed for using Travis builds. I believe using their public builds is free. This will help identify if tests are passing or failing.

Reply:
Main problem with Travis is that we have internal infrastructure for e2e test and we already using Jenkins. Though we're planning to integrate Jenkins builds with github and PRs.

Reply:
Ah, gotcha! I will strip it. It would be great to publicly see whether tests are passing here though.